L1Trigger/Phase2L1ParticleFlow/test/BuildFile.xml

```diff
@@ -0,0 +1,9 @@
+<bin name = "testOutputFiles" file="testOutputFiles.cpp">
+	<flags TEST_RUNNER_CMD="cd $(CMSSW_BASE)/src/L1Trigger/Phase2L1ParticleFlow/test; testOutputFiles Phase2L1ParticleFlowOutput.root trackerRegion_alltracks_sectors_2x9_TTbar_PU200.dump trackerRegion_alltracks_sectors_2x9_TTbar_PU200.coe 0 0 9 -3 0 3"/>
```

Comment:
the input files for this test should be put in the associated data area

Reply:
uhm, this is not supposed to be run in the standard workflow.
if it creates problems for the integration, I'd rather just remove it and ask the person using it to put it elsewhere

Reply:
as far as I know, scram always runs all defined unit tests
@smuzaffar is this correct?

Reply:
added `<Flags NO_TESTRUN="1"/>` to the BuildFile